icicle-compiler/test/cli/zebra/run

```diff
@@ -6,7 +6,7 @@ echo "Separate compilation + Zebra"
 
 common_args="--snapshot 2010-01-01"
 dict=test/cli/zebra/dictionary.toml
-in=test/cli/zebra/input.zbin2
+in=test/cli/zebra/input.zbin3
```

Comment:
I think, according to what @jystic said, you should be able to test with both zbin2 and zbin3 now?